cmd/server/main.go

```diff
@@ -124,15 +123,6 @@ func initConfig() error {
 		return err
 	}
 
-	ServerConf.FlavourMap = make(map[string]schema.Flavour)
```

Comment:
Add init sql to `installer/database/paddleflow.sql`

Reply:
done